solidity/contracts/converter/ConverterUpgrader.sol

```diff
@@ -64,8 +74,24 @@ contract ConverterUpgrader is IConverterUpgrader, ContractRegistryClient {
      *
      * @param _version old converter version
      */
-    function upgrade(uint16 _version) public override {
-        upgradeOld(IConverter(msg.sender), bytes32(uint256(_version)));
+    function upgrade(uint16 _version) external override {
+        upgrade(IConverter(msg.sender), _version);
+    }
+
+    /**
+     * @dev upgrades an old converter to the latest version
+     * will throw if ownership wasn't transferred to the upgrader before calling this function.

```

Comment:
IMO: 'will throw' ==> 'reverts' (consistent with our present simple tense throughout the contracts).